csaf_2.0/guidance/sbom-matching-interface.md

```diff
@@ -0,0 +1,91 @@
+| `PartialStringMatch`             | `0.50`           | Indicates that a string property (e.g., the vendor name) of the product partially matches the component's string property.                                                                                                                                          |
+| `MatchPackageNoVersion`          | `0.70`           | Indicates that the package name of the product matches the component's package name, but no version was specified. So in theory, all versions that are in the SBOM could be a match.                                                                                |
+| `DifferentSources`               | `0.90`           | Indicates that the information comes from different sources (e.g., matching a vendor in a CPE to a vendor specified in a CSAF product tree category. This can be used to "multiply" the matching confidence with this value to adjust it for the different sources. |
+| `CaseInsensitiveIgnoreDashMatch` | `0.90`           | Indicates a (string-based) match that is case-insensitive and ignores dashes. This is a relatively high match value, but not as high as a `CaseInsensitiveMatch`.                                                                                                   |
+| `CaseInsensitiveMatch`           | `0.95`           | Indicates a (string-based) match that is case-insensitive. This is a high match value, but not as high as a `DefiniteMatch`.                                                                                                                                        |
+| `DefiniteMatch`                  | `1.00`           | Indicates a definite match. This is the highest possible match value. This should be used if two properties match exactly -- either lexically or by rules defined in a specification. For example if one CPE matches another, this will be a `DefiniteMatch`.       |
+
+## Matching Properties
+
+Since much different information are available both on the security advisory and the SBOM document or node, we need a way to structure information to correlate between them. For example, the SBOM node might not directly indicate the version of the product, but it might specify a CPE which includes the version. On the other hand, the CSAF advisory might indicate the version and product name of the product, but not a CPE. In this case we want the matching to be able to correlate the information based on the properties (product name and version) contained in different places (CPE, categorized values). If we would just match based on a CPE, we would not be able to generate a match in this example.
+
+Therefore, we define an (initial)  set of properties that can be used to establish a match. In general, it is important to know that a "property" holds a certain set of meta-data:
+- `name`: The name of the property
```

Comment:
Nit.Again, suggest a definition list, as we define terms like so:
```

`name`
:    The name of the property.
 
`value`
:    The value of the property.
This can be a string or a more specialized structure, such as a product version or identifier.

`source`
:    The source of the property.
Currently, extraction of the property from the CSAF/SBOM structure as well as from CPE and PURL identifiers contained in them are supported.

```